bundles/org.openhab.binding.nanoleaf/src/main/resources/OH-INF/i18n/nanoleaf_de.properties

```diff
@@ -1,9 +1,9 @@
 binding.nanoleaf.name = Nanoleaf Binding
-binding.nanoleaf.description = Binding für die Integration des Nanoleaf Light Panels (Ein mit dem Controller verbundenes Paneel)
```

Comment:
Please remove the changes in German translation. Changes must be provided through Crowdin after the PR has been merged. See https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations.